### PR TITLE
📃 Further flow, code snippets (in re #651)

### PR DESCRIPTION
**Pull request by JFWooten4**
As for the issuer / issuing syntax, I believe it's best practice to keep gerunds out of variable names.

Some more to go, just saving this progress; requesting any feedback on granularity.

### CODE REVIEW

**Comment by briwylde08** [comment]
Amazing! Thank you so much! Our team will review and get back to you with questions or approval. Friendly reminder that I gave reviewers three weeks to review submissions but hopefully we'll be able to get to it sooner [🎉](https://emojipedia.org/party-popper)

Solves: https://github.com/stellar/stellar-docs/issues/651

File: docs/issuing-assets/how-to-issue-an-asset.mdx
Comment (ElliotFriend) [inline docs/issuing-assets/how-to-issue-an-asset.mdx:328]: It would probably be a good idea in all the code examples in this step to add a comment in the snippet, something like...

```python
transaction = (
    # We're using TransactionBuilder(...) as a short-hand here to show that these operations can be "chained" together
    TransactionBuilder(...)
    .append_payment_op(
```

Comment (ElliotFriend) [reply docs/issuing-assets/how-to-issue-an-asset.mdx:328]: Also, just realizing now that your python is only indented by 2 spaces. We want that to be 4.

File: docs/issuing-assets/how-to-issue-an-asset.mdx
Comment (ElliotFriend) [inline docs/issuing-assets/how-to-issue-an-asset.mdx:145]: Missing a closing parenthesis `)` at the end of this line

**Review by ElliotFriend — COMMENTED** [review]
Lookin' pretty good! Couple of suggestions and questions here, but I think it's close!

Thanks for the contribution!!

File: docs/issuing-assets/how-to-issue-an-asset.mdx
Comment (JFWooten4) [reply docs/issuing-assets/how-to-issue-an-asset.mdx:282]: In Solidity e.g., you may adjust the number of tokens issued by a decimal scaling factor:
```javascript
pragma solidity ^0.8.0;

import "@openzeppelin/contracts/token/ERC20/ERC20.sol";

contract MyToken is ERC20 {
    constructor(uint256 initialSupply) ERC20("MyToken", "MTK") {
        // The initial supply is multiplied by 10^18 to account for 18 decimal places
        _mint(msg.sender, initialSupply * 10 ** uint(decimals()));
    }
    
    function decimals() public view virtual override returns (uint8) {
        return 18;
    }
}
```

I thought it might be relevant to note that you don't need to scale by Stellar's [minimum increment](https://developers.stellar.org/docs/learn/fundamentals/stellar-data-structures/assets#amount-precision).

What do you think?

File: docs/issuing-assets/how-to-issue-an-asset.mdx
Comment (JFWooten4) [reply docs/issuing-assets/how-to-issue-an-asset.mdx:67]: I'm partial to placing Python first for readability amidst new devs. 💭

global-py-dev-starter-small

Here is the updated image with the modern Python logo and elements invoking global developer inclusion.

Also, I noticed that existing python docs use `snake_case` even though all other languages employ `camelCase` (as is best practice in [the ecosystem](https://github.com/stellar/stellar-core/blob/master/CONTRIBUTING.md#code-style)). 🤔

While I appreciate that the underlying community library uses snake case, it's my opinion that we should consider camel case across the board for Python examples too, in an effort to lower the bar for developers by legitimizing the language. 🚀

Comment (ElliotFriend) [reply docs/issuing-assets/how-to-issue-an-asset.mdx]: I'm a big fan of Python, so I'm there with you when it comes to putting python first in the examples. But, our ecosystem is a lot more JS-heavy, so I don't think that's quite the move for us.

---

I'd probably push back on `camelCase` in python examples, though. Yes, most other languages do like to use camel-case, but it's a **really** well-established pattern within the broader Python community to `snake_case_just_about_everything`. Most Python linters enforce snake-case, by default, even. I think working within established patterns is probably our best bet for any language example.

From my point of view, with any code example, the goals should be:

1. Non-proficient devs in that language can pretty well understand the logic, flow, etc. of the concept being exemplified, even if they don't get the finer points of that specific language.
2. Proficient devs in that language can quickly get up-to-speed on the Stellar-specific things about the SDK and/or concept without having to interrupt their train of thought with thoughts/questions about the package itself (i.e., "why did they do it _this_ way? everything in `language` is usually done _this other_ way...")

File: docs/issuing-assets/how-to-issue-an-asset.mdx
Comment (ElliotFriend) [reply docs/issuing-assets/how-to-issue-an-asset.mdx]: Oooh, so you're basically saying something like "unlike Solidity, the decimal precision is already baked-in when using the `payment` operation"? I like it. Sounds like useful information for someone coming from EVM chains.

(I'm a dummy at Solidity, so I might be completely missing the point lol)

File: docs/issuing-assets/how-to-issue-an-asset.mdx
Comment (ElliotFriend) [reply docs/issuing-assets/how-to-issue-an-asset.mdx]: That's a really good point you bring up. Our [JS prettier config](https://github.com/stellar/product-conventions/blob/master/packages/prettier-config/prettier.config.js) uses 2 spaces, but for Python we don't really have anything stating that 4 spaces are required. Again, just falling back to the norms and conventions of the broader Python community there.

We've got on the (long) roadmap that we need to come up with some of those codified, stated, enforced choices. In the meantime, I don't have a strong enough opinion to let python tab-width block a PR from being merged lol.

**Review by ElliotFriend — APPROVED** [review]
Great work, @JFWooten4! Looks like you've got a small conflict now. Once that's resolved, we can get this baby merged!

Thanks again!!!

**Comment by JFWooten4** [comment]
> Looks like you've got a small conflict now. 

What do you mean exactly? I think it's just this file. Should satisfy basic formatting needs. 👨‍💻

**Comment by ElliotFriend** [comment]
From what I can tell there's a conflict because the `docs/issuing-assets/how-to-issue-an-asset.mdx` file has been updated since you started working on this PR. You can merge the `main` branch back into your `patch-1` branch, and that should help you resolve whatever conflict is triggering the alert.

**Comment by briwylde08** [comment]
@JFWooten4! Hi! I've been trying to figure out how to do this locally but it doesn't seem to be working. Have you tried to resolve the conflict(s) yet?

**Comment by briwylde08** [comment]
Yay, thanks for fixing this!

**Pull request merged**
